Cluster Programs

```diff
@@ -0,0 +1,57 @@
+Wednesday 13/06/2018
+
+I begin running the SAS programs.
+
+NE Control_1 
+I made a copy of the original and I'm working on the file called NE Control_1 JV
+
+I change the name of the directory in the firts line to:
+G:\MAX-Filer\Research\Labs-kbuzard-S18 
+then I run the entire program one step at a time.
+Started runing at 10.40 am ended at 10.53 am. If I run the entire program it takes less than 3 minutes. 
+I added the Procedure Print the variable citations at the end of the program to see the result.
+Correctly print a table. 
+
+After running the hole program step by step, I start checking it by printing some of the variables, 
+the program print the tables but it take beetwen 10-15 min per table, if it takes longer than that I cancel the printing. 
+
+The next step should be to transpone the columns so the table looks like the one in the paper we are using.
+I assume this is done in the later programs. 
+
+The program has some value that are not referenced in the code. 
+
+Line 124, col 34. I got this message "WARNING: The variable appyear in the DROP, KEEP, or RENAME list has never been reference"
+Line 179, col 25. I got this message "WARNING: The variable o_gday in the DROP, KEEP, or RENAME list has never been referenced".
+Line 179, col 25. I got this message "WARNING: The variable o_gmonth in the DROP, KEEP, or RENAME list has never been referenced".
```

Comment:
Please investigate these warnings. I never did. They either indicate a problem, or that we can likely clean up the code a little bit. I had to implement a change to get different date variables in the programs so that we could do matching based on a 182-day window (this is in the second program), and it required a different set of date variables, but I don't remember whether these ones like o_gday are the original date variables or the new ones that were needed to deal with the 182-day window. I don't know how obvious it will be by looking at the code, but you may be able to figure it out by looking at older programs that don't do the 182-day window. Those restrictions on the matching are all just above where the 20 iterations of the matching start in program 2